src/chapter-03/template.yaml

```diff
@@ -18,12 +18,12 @@ Resources:
       Architectures:
         - arm64
       Events:
-        Api:
+        CreateShortUrl:
           Type: HttpApi
           Properties:
             Path: /
```

Comment:
Should this one be `/links` (based on your diagram)? It also makes it a bit more RESTful